attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -0,0 +1,217 @@
+    1 => non-anonymous-identity,
+    2 => account-multisig-token,
+    3 => non-anonymous-identity,
+}
+; end::types[]
+
+; tag::feature[]
+; The feature argument when creating a new account.
+multisig@account-feature@argument = {
+    ; Threshold of number of approvals needed by default to execute a transaction.
+    ; This number can be overridden to be greater than the default per transaction,
+    ; but cannot be overridden to be less.
+    ; This number includes the submitter of a transaction, meaning that the value of
+    ; this field is always at least 1 (passing `0` will fail creating the account).
+    ; If this value is greater than the number of people with the `canApprove` role,
+    ; the server MAY still create the account.
+    ; By default this is the number of identities with the `canApprove` role.
+    ? 0 => uint,
+
+    ; The time (in seconds) a transaction would automatically expire. The server is
+    ; allowed to clamp this value, or refuse to create an account if this is too
+    ; large.
+    ? 1 => uint,
+
+    ; If true, transactions will be executed automatically when the threshold is
+    ; met.
+    ; By default the `account.multisigExecute` must be called by an owner or the
+    ; creator of the transaction.
+    ; Server MAY return an error if this is true and it doesn't support automatic
+    ; execution.
+    ? 2 => bool,
+}
+; end::feature[]
+
+; tag::setDefaults[]
+; Update default values for new transactions.
+; The server MUST return an error if the sender does not have the `owner` role.
+account.multisigSetDefaults@param = {
+    ; A new default threshoold.
+    ? 0 => uint,
+
+    ; A new time (in seconds) to automatically expire transactions.
+    ? 1 => uint,
+
+    ; If true/false, update whether transactions are automatically executed.
+    ; Server MAY return an error if it doesn't support automatic execution.
+    ? 2 => bool,
+}
+
+account.multisigSetDefaults@return = ()
+; end::setApproverCount[]
+
+; tag::submitTransaction[]
+; Submit a transaction to be executed when the threshold is reached.
+account.multisigSubmitTransaction@param = {
+    ; A memo associated with the transaction.
+    ? 0 => tstr,
+
+    ; The transaction to send. Either a ledger transaction object, or a bytestring that's
+    ; a request message with limited fields.
+    ; Servers are allowed to refuse a submitted message and return an error.
+    1 => account-multisig-transaction,
+
+    ; A number of approvers for this specific transaction. The server MUST return an
+    ; error if this value is passed but the submitter does not have the `owner` role.
+    ? 2 => uint,
+
+    ; Whether to execute this transaction automatically when the threshold is met.
+    ; The server MUST return an error if this value is passed but the submitter
+    ; does not have the `owner` role, or if automatic execution is not supported.
+    ? 3 => bool,
+
+    ; The time (in seconds) this transaction would automatically expire. The server is
```

Comment:
Do you want expiration precision in msec? Like "this transaction expire in 3 hours 24 minutes 15 seconds and... 289 msec"? I think that's a bit much.

I went with seconds because every time value so far are in seconds.